database/crdb.ksy

```diff
@@ -0,0 +1,165 @@
+meta:
+  id: crdb
+  title: GNU/Linux wireless regulation database
+  license: GPL-2.0
```

Comment:
I believe this is somewhat more complicated. It's amalgam of ISC-licensed data, and CopyLeft-Next-licensed code.

Reply:
You are right! I thought that everything on kernel.org is under gpl.